cfme/tests/webui/test_splitter.py

```diff
@@ -1,20 +1,44 @@
 import pytest
-from cfme.web_ui.splitter import pull_splitter_left, pull_splitter_right
 from xml.sax.saxutils import quoteattr, unescape
+
+from cfme.base.ui import Server
+from cfme.cloud.instance import Instance
+from cfme.infrastructure.config_management import ConfigManager
+from cfme.infrastructure.datastore import Datastore
+from cfme.infrastructure.pxe import ISODatastore
+from cfme.infrastructure.virtual_machines import Vm
+from cfme.intelligence.chargeback import ComputeRate
+from cfme.intelligence.reports.reports import CustomReport
+from cfme.services.myservice import MyService
+from cfme.web_ui.splitter import pull_splitter_left, pull_splitter_right
 from utils import version
+from utils.appliance.implementations.ui import navigate_to
 from utils.blockers import BZ
+
+
 LOCATIONS = [
-    "control_explorer", "automate_explorer", "automate_customization", "my_services",
-    "services_catalogs", "services_workloads", "reports", "chargeback", "clouds_instances_no_tree",
-    "infrastructure_virtual_machines_no_tree", "infrastructure_pxe", "configuration",
-    "infrastructure_datastores_no_tree", "infrastructure_config_management", "utilization",
-    "bottlenecks",
+    "control_explorer", "automate_explorer", "automate_customization", MyService,
+    "services_catalogs", "services_workloads", CustomReport, ComputeRate, Instance, Vm,
+    ISODatastore, Server, Datastore, ConfigManager, "utilization", "bottlenecks",
     "infrastructure_networking"]
 
 pytestmark = [pytest.mark.parametrize("location", LOCATIONS), pytest.mark.uncollectif(lambda
     location: location == "infrastructure_networking" and version.current_version() < '5.7')]
 
 
+def nav_to(location):
+    if isinstance(location, basestring):
+        pytest.sel.force_navigate(location)
+    else:
+        dest = 'All'
+        if location is Vm:
+            dest = 'VMsOnly'
+        elif location is Server:
+            dest = 'Dashboard'
+
```

Comment:
Is there a fallback if the destination doesn't exist for a location object? Ex if VMsOnly isn't implemented, do we want to fall back to Dashboard, or skip the test?